ember-client/shared/src/main/scala/org/http4s/ember/client/EmberClientBuilder.scala

```diff
@@ -128,6 +141,16 @@ final class EmberClientBuilder[F[_]: Async] private (
   def withUnixSockets(unixSockets: UnixSockets[F]) =
     copy(unixSockets = Some(unixSockets))
 
+  def withHttp2 = copy(enableHttp2 = true)
+  def withoutHttp2 = copy(enableHttp2 = false)
+
+  def withPushPromiseSupport(
```

Comment:
A scaladoc on this signature would go a long way.  I'm not sure what I'd call this with.